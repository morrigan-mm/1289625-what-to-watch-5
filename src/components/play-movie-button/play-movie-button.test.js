import React from "react";
import renderer from "react-test-renderer";
import PlayMovieButton from "./play-movie-button";

describe(`PlayMovieButton`, () => {
  it(`Should render correctly`, () => {
    const tree = renderer
      .create(
          <PlayMovieButton />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
