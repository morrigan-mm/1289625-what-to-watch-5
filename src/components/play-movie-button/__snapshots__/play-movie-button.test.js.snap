// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayMovieButton Should render correctly 1`] = `
<button
  className="btn btn--play movie-card__button"
  type="button"
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      href="#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</button>
`;
